stdlib/public/Glibc/CMakeLists.txt

```diff
@@ -1,46 +1,59 @@
 set(sources
-    Glibc.swift
-    Misc.c
+  Glibc.swift
+  Misc.c
 )
 
-set(output_dir "${SWIFTLIB_DIR}/glibc")
+# When cross-compiling the stdlib on Unix platforms, we'll need a separate
+# glibc for each target.
+foreach(SDK ${SWIFT_SDKS})
```

Comment:
Aha! And that, in fact, was the very problem with the OS X CI. Thanks for pointing this out.